doc/source/science_guide/sg_dynamics.rst

```diff
@@ -505,7 +505,7 @@ where
    \Delta = \left[D_D^2 + {e_f^2\over e_g^4}\left(D_T^2 + D_S^2\right)\right]^{1/2}.
    :label: Delta
 
-When the deformation :math:`\Delta` tends toward zero, the viscosities tend toward infinity. To avoid this issue, :math:`\Delta` needs to be limited and is replaced by :math:`\Delta^*` in equation :eq:`zeta`. Two methods for limiting :math:`\Delta` (or for capping the viscosities) are available in the code. If the namelist parameter ``capping`` is set to 1., :math:`\Delta^*=max(\Delta, \Delta_{min})` :cite:`Hibler79` while with ``capping`` set to 0., the smoother formulation  :math:`\Delta^*=(\Delta + \Delta_{min})` of :cite:`Kreyscher00` is used. 
+When the deformation :math:`\Delta` tends toward zero, the viscosities tend toward infinity. To avoid this issue, :math:`\Delta` needs to be limited and is replaced by :math:`\Delta^*` in equation :eq:`zeta`. Two methods for limiting :math:`\Delta` (or for capping the viscosities) are available in the code. If the namelist parameter ``capping_method`` is set to ``hibler``, :math:`\Delta^*=max(\Delta, \Delta_{min})` :cite:`Hibler79` while with ``capping_method`` set to ``kreyscher``, the smoother formulation  :math:`\Delta^*=(\Delta + \Delta_{min})` of :cite:`Kreyscher00` is used. 
```

Comment:
Can we use the labels 'max' or 'sum' for capping_method instead of people's names?  It's more intuitive, and I've always tried to avoid naming things after individual people except when it's unavoidable.

Reply:
Happy to make that change, not a problem.  What strings do we want to use instead?  @JFLemieux73 @eclare108213  Just let me know and I'll make the change.

Reply:
I suggest 'max' instead of Hibler and 'sum' instead of Kreyscher

Reply:
I changed the implementation to "max" and "sum".  I also added some tests (alt07) to test capping="sum" and visc_method="avg_zeta" and a different ndte.  We can continue to update alt07 as we work towards better coverage of the dynamics options.